server/user/user.controller.js

```diff
@@ -23,13 +24,22 @@ const inputAttrs = ['email', 'role', 'firstName', 'lastName'];
 const createFilter = q => map(['email', 'firstName', 'lastName'],
   it => ({ [it]: { [Op.iLike]: `%${q}%` } }));
 
+const addLastActiveProperty = user => {
```

Comment:
This method is replacing user's `lastActive` if it is present in the tracker. Please rename it to describe that behaviour or consider `processOutput` as name.